splitio_platform_interface/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Now that I see this file, check if you need a GHA script like this one https://github.com/splitio/javascript-client/blob/development/.github/workflows/update-license-year.yml
to automatically update license files.